packages/react-components/src/Popup.tsx

```diff
@@ -2,6 +2,30 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Popup } from 'semantic-ui-react';
+import { Popup as PopupBase, StrictPopupProps } from 'semantic-ui-react';
+import React from 'react';
+import styled from 'styled-components';
 
-export default Popup;
+function Popup({ content, trigger, className }: StrictPopupProps): React.ReactElement {
+    return <PopupBase
+      content={content}
+      trigger={trigger}
+      className={className}
+    />
+}
+
+export default styled(Popup)`
+    background-color: #1B1C1D !important;
```

Comment:
If possible, try to not use `!important`, which has a quite high css ruling priority and not easy to be overrided by potential future styles.

Reply:
yes I though this aswell, seems like with styled components it's acceptable because they have unique stylesheets (like your below comment mentions?)